### ISSUE TITLE
MySQL version < 5.7.2 - Triggers have null created value and throw errors when parsing

### ISSUE THREAD

**Issue by andrewmcfaul** [k0kubun/sqldef#157]
This took a while to understand but in `mysqldef.go: line 91` the results of the `SHOW TRIGGERS` command are being scanned in as strings, however in MySQL < 5.7.2 the `created` value is always `NULL`. [See Documentation](https://dev.mysql.com/doc/refman/5.7/en/show-triggers.html)

This causes the error:
> Error on DumpDDLs: sql: Scan error on column index 5, name "Created": converting NULL to string is unsupported

## --export output
Crashes with error

## Input SQL
Any table with any trigger created under MySQL < 5.7.2 environment.

## Current output
Crashes with error

## Expected output
Should parse the trigger with NULL created value

**Comment by andrewmcfaul** [comment]
Hi, thanks for having a look.

We are actually using MySQL 5.7.34, but the triggers were created when we were using 5.6.x.

I have just checked locally and you should be able to reproduce the issue like this.

Create a directory on your host machine to hold data, we are going to bind to it and share it between two docker instances.
I am using '/mysql_test/data'

**Create container for old version with data directory on the host machine**
`docker pull mysql/mysql-server:5.6.51`
`docker run --name=mysql_test -v /mysql_test/data:/var/lib/mysql -d mysql/mysql-server:5.6.51`


**Reset the password on the root user account, otherwise we won't be able to upgrade later.**
`docker exec -it mysql_testing_57 mysql -uroot -p`
and then `SET PASSWORD = PASSWORD('some_password');`

**Set up schema with trigger**
`docker exec -it mysql_testing_57 mysql -uroot -p`
Enter password, then run

```
CREATE TABLE `test` (   `id` int(11) NOT NULL AUTO_INCREMENT,   PRIMARY
KEY (`id`) ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

Then 

`CREATE DEFINER=`root`@`localhost` TRIGGER trig_test AFTER INSERT ON test FOR EACH ROW UPDATE test SET id = id + 1;`

**Now stop the container.**
`docker stop mysql_test`

**Create new container with MySQL 5.7.34**
`docker pull mysql/mysql-server:5.7.34`
`docker run --name=mysql_testing_57 --mount type=bind,src=/mysql_test/data,dst=/var/lib/mysql -d mysql/mysql-server:5.7.34`

**Run the upgrade script**
`docker exec -it mysql_testing_57 mysql_upgrade -uroot -p`
Enter the password that you set on the old container.

**Now run mysqldef and you should see the issue.**
You can also run `SHOW TRIGGERS` and you should see that created is still set to `NULL`

**Comment by k0kubun** [comment]
Thank you. I confirmed it becomes NULL with the provided steps.

I fixed it in https://github.com/k0kubun/sqldef/commit/1d75663e633b7f70a34aee6b443eba083bad101f. It'll be released as v0.11.9. Unfortunately, the maintenance cost of testing the procedure is too high, so I didn't add a test for this. So the service level of DDLs migrated from 5.6 is not as high as the ones created in 5.7. I hope you'll replace such triggers with triggers defined in 5.7 so that you will not be impacted when it's broken again as we don't have tests for it.
